Makefile

```diff
@@ -30,6 +30,9 @@ kubectl get service $(1) -n $(NAMESPACE) | grep $(1) | awk '{print $$4 ":" $$5}'
 endef # get_service
 VERIFY_CLUSTER = _verify_cluster
 endif # TARGET
+define delete_service_pods
+kubectl delete pods -n $(NAMESPACE) $(shell kubectl get pods -n $(NAMESPACE) | grep assisted-service | awk '{ print $$1 }')
```

Comment:
you can try 
```
kubectl rollout restart deployment assisted-service -n $(NAMESPACE)
```

Reply:
If you use `rollout restart` consider renaming  `delete_service_pods` to `restart_service_pods`